README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/mozilla/api.webmaker.org.svg)](https://travis-ci.org/mozilla/api.webmaker.org)
+
 # api.webmaker.org
```

Comment:
Might even want to remove the roadmap / platform stuff from here too as we are tracking those things in issues.